13/umbraco-commerce/how-to-guides/add-item

```diff
@@ -0,0 +1,192 @@
+---
+description: How-To Guide to add an item to your cart.
+---
+
+# Add item to Cart
+
+To add an item to the cart, you need to set up Umbraco with a store and add the relevant properties to allow the store to interact with Umbraco. Learn more by following the [Getting started with Umbraco Commerce: The Backoffice tutorial](../tutorials/getting-started-with-commerce).
+
+You will need the front end to be set up to allow an item to be added to the cart. This can be done by adding a button to the front end to call the Action to add the item to the cart.
+
+Create a new Document Type with the template. Call it **Product Page** with the following property aliases: `productTitle`, `productDescription`, `price`, `stock`.
```

Comment:
Which Property Editors should be used for these properties?

Reply:
@sofietoft Ive added those, sorry not sure if the correct styling was applied as for me the Preview feature doesnt seem to be working :-(

Reply:
- `productTitle`: text string
- `productDescription`: text string
- `price`: Umbraco Commerce: Price
- `stock`: Umbraco Commerce: Stock